lensless/benchmark.py

```diff
@@ -0,0 +1,192 @@
+"""
+
+Download subset from here: https://drive.switch.ch/index.php/s/vmAZzryGI8U8rcE
+Or full dataset here: https://github.com/Waller-Lab/LenslessLearning
+
+To use integrated test function rename the downloaded folder 'DiffuserCam_Mirflickr' and place it inside the data folder.
+"""
+
+import glob
+import os
+import pathlib as plib
+from datetime import datetime
+from lensless.io import load_psf
+import numpy as np
+from tqdm import tqdm
+
+try:
+    import torch
+    from torch.utils.data import Dataset, DataLoader
+    from torch.nn import MSELoss, L1Loss
+    from torchmetrics import StructuralSimilarityIndexMeasure
+    from torchmetrics.image import lpip, psnr
+except ImportError:
+    raise ImportError("Torch and torchmetrics are needed to benchmark reconstruction algorithm")
+
+
+class RealDataset(Dataset):
+    """
+    Dataset consisting on mesure images with both a lensless and a lensed setup.
```

Comment:
"Dataset consisting of lensless and corresponding lensed image."

perhaps we use this object for both measured and simulated datasets?